### DISCUSSION TITLE
Location error???

### DISCUSSION THREAD

**Discussion by takedakoichi** [community/community#52148] [GitHub Education]
I am a student in Japan.
I registered using the e-mail address provided by my school.
My university name is automatically detected in the application form.
I have taken a translated photo of my student ID card.
I am currently on campus, so my location should be correct.

However, I cannot seem to verify my student ID.
I have tried several times and get the following message back.

・The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

・You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

**Comment by samus-aran** [comment]
Hi @takedakoichi 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by samus-aran** [reply]
Hi @takedakoichi 

We have updated your selected school with the correct details and ask for you to re-apply again. Let us know how you get on 👍

**Reply by takedakoichi** [reply]
Success.Thank you.

**Comment by ghost** [comment]
I have the same problem

**Comment by rajeevkumar504** [comment]
I also face something similar problem.

**Comment by makoto1119** [comment]
me too.
how can i do that for fix the problem？
